cmd/kubeadm/app/phases/certs/constants/constants.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package constants
+
+const (
+	CACertAndKeyBaseName = "ca"
+	CACertName           = "ca.crt"
+	CAKeyName            = "ca.key"
+
+	APIServerCertAndKeyBaseName = "apiserver"
+	APIServerCertName           = "apiserver.crt"
+	APIServerKeyName            = "apiserver.key"
+)
```

Comment:
The are used elsewhere, and I don't want to import the full phase package.

We might want to consider a toplevel `cmd/kubeadm/app/constants` package though, sounds better?